sentry_sdk/sessions.py

```diff
@@ -16,17 +16,18 @@
     from typing import Dict
     from typing import List
     from typing import Generator
+    from typing import Union
 
 
 def is_auto_session_tracking_enabled(hub=None):
-    # type: (Optional[sentry_sdk.Hub]) -> bool
+    # type: (Optional[sentry_sdk.Hub]) -> Union[bool, None]
     """Utility function to find out if session tracking is enabled."""
     if hub is None:
         hub = sentry_sdk.Hub.current
     should_track = hub.scope._force_auto_session_tracking
     if should_track is None:
-        exp = hub.client.options["_experiments"] if hub.client else {}
-        should_track = exp.get("auto_session_tracking")
+        client_options = hub.client.options if hub.client else {}
+        should_track = client_options.get("auto_session_tracking", True)
```

Comment:
you can use `["auto_session_tracking"]` since we guarantee a default elsewhere already